### PR TITLE
Jesus workshops2

### PR DESCRIPTION
**Pull request by JesusGitHubOracle**
Worksop page plus two pdf files

### CODE REVIEW

File: data-platform/core-converged-db/workshops/README.md
Comment (AlexanderHodicke) [inline data-platform/core-converged-db/workshops/README.md:1]: groups of assets should be under product not sub-area pages

Comment (JesusGitHubOracle) [reply data-platform/core-converged-db/workshops/README.md:1]: Hi Alex - understood. I'll fix this

File: data-platform/core-converged-db/workshops/WORKSHOP_MAA_Part3_Sharding_ENG_v1.1.pdf
Comment (AlexanderHodicke) [inline data-platform/core-converged-db/workshops/WORKSHOP_MAA_Part3_Sharding_ENG_v1.1.pdf:1]: year and oracle confidential to the public

Code review by @oheimburger 

Possibly containing IP addresses!

**Review by AlexanderHodicke — CHANGES_REQUESTED** [review]

File: data-platform/core-converged-db/workshops/WORKSHOP_MAA_Part3_Sharding_ENG_v1.1.pdf
Comment (oheimburger) [reply data-platform/core-converged-db/workshops/WORKSHOP_MAA_Part3_Sharding_ENG_v1.1.pdf:1]: Page 6: Change the public IPs into non-routable IPs like 192.168.0.1.
Page 8: wget URL should be replaced by "ask instructor for the URL"
Starting at page 18 replace all occurrences of passwords, like Ora_DB4U, app_schema,TEZiPP4MsLLL,etc., with '<your_password>'
Page 25: wget URL should be replaced by "ask instructor for the URL"
Page 25: Remove all IP numbers and hostnames from the wget output
Page 26: wget URL should be replaced by "ask instructor for the URL"
Starting at Page 27: Password TEZiPP4MsLLL should be replaced by a placeholder.

File: data-platform/core-converged-db/workshops/JSON.PERF_v1_ENG.pdf
[the remark this answers is not in the document]
Comment (oheimburger) [reply data-platform/core-converged-db/workshops/JSON.PERF_v1_ENG.pdf:1]: Replace all passwords with a placeholder everywhere, like soe/soe.

**Review by oheimburger — CHANGES_REQUESTED** [review]

**Pull request closed without merging**